app/test_main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+from unittest import mock
+import pytest
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "valid_url, connection, access",
+    [
+        pytest.param(True, True, "Accessible",
+                     id="Test can access. All valid"),
+        pytest.param(True, False, "Not accessible",
```

Comment:
Add an error message. It's good when the user understands what the problem is. Fix everywhere

Reply:
was added more detailed id